guides/common/assembly_basic-content-management-workflow.adoc

```diff
@@ -0,0 +1,29 @@
+include::modules/con_basic-content-management-workflow.adoc[]
```

Comment:
Perhaps this assembly should be included in the `master.adoc`?

Reply:
@maximiliankolb And this one

Reply:
Fixed. I had (by mistake) included the concept instead of the assembly.